packages/common/src/utils/item.ts

```diff
@@ -45,7 +45,7 @@ export const getItemId = (item: Field | AdditionalMetric | TableCalculation) =>
     isField(item) || isAdditionalMetric(item) ? fieldId(item) : item.name;
 
 export const getItemLabel = (item: Field | TableCalculation) =>
-    isField(item) ? `${item.tableLabel} ${item.label}` : item.displayName;
```

Comment:
Changing this method is going to change every label in the app that shows the table name + the field name. Is that what @PriPatel meant? 
If we want to only change the chart legend/labels/tooltips we need to be more precise with our fix

Reply:
it's not affecting `tables` and `big numbers`

Reply:
Should only affect chart legend / label stuff 👍

Reply:
> it's not affecting `tables` and `big numbers`

@yasmin-e It is affecting a lot of features and areas of the app including the table and big number.

![Captura de ecrã 2023-02-14, às 09 54 16](https://user-images.githubusercontent.com/9117144/218700949-aae3fffb-547b-49f2-9aea-2b655d6a5c71.png)

Reply:
only place where it won't be reflected it in this autocomplete field because the component is being used in many places, however once the field is selected it appears on the chart without the table name 

<img width="376" alt="Screenshot 2023-02-14 at 11 22 33" src="https://user-images.githubusercontent.com/67699259/218707885-a998a707-b93d-4318-8741-2d77d7e40aca.png">